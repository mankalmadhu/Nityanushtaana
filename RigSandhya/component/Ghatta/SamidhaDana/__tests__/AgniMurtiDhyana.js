import React from 'react';
import ಅಗ್ನಿಮೂರ್ತಿಧ್ಯಾನ from '../AgniMurtiDhyana';
import Renderer from 'react-test-renderer';

jest.mock('../../Ghatta.vinyasa');
jest.mock('../../../Vishaya/Vishaya',()=>'ವಿಷಯ');
jest.mock('../../../Vishaya/Sheershike',()=>'ಶೀರ್ಷಿಕೆ');
jest.mock('../../../Vishaya/Purvanga',()=>'ಪೂರ್ವಾ೦ಗ');
jest.mock('../../../Vishaya/Viniyoga',()=>'ವಿನಿಯೋಗ');

test("ಅಗ್ನಿಮೂರ್ತಿಧ್ಯಾನ", () => {
   expect(Renderer.create(
    <ಅಗ್ನಿಮೂರ್ತಿಧ್ಯಾನ />
  ).toJSON()).toMatchSnapshot();

});