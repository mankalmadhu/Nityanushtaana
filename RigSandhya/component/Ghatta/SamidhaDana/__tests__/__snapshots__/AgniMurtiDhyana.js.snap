// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ಅಗ್ನಿಮೂರ್ತಿಧ್ಯಾನ 1`] = `
<View>
  <ವಿಷಯ
    ವಿನ್ಯಾಸ="ಶೀರ್ಷಿಕೆ"
  >
    ಅಗ್ನಿಮೂರ್ತಿ ಧ್ಯಾನ
  </ವಿಷಯ>
  <ವಿಷಯ
    ವಿನ್ಯಾಸ="ಪೂರ್ವಾ೦ಗ"
  >
    ಚತ್ವಾರಿಶೃ೦ಗೇತಿ ವಾಮದೇವೋ ಜಾತವೇದಾಗ್ನಿಃ ತ್ರಿಷ್ಟುಪ್ ಅಗ್ನಿಮೂರ್ತಿ ಧ್ಯಾನೇ ವಿನಿಯೋಗಃ ।
  </ವಿಷಯ>
  <ವಿಷಯ
    ವಿನ್ಯಾಸ="ವಿನಿಯೋಗ"
  >
    ಚ॒ತ್ವಾರಿ॒ ಶೃಙ್ಗಾ॒ ತ್ರಯೋ᳚ ಅಸ್ಯ॒ ಪಾದ॒ ದ್ವೇ ಶೀ॒ರ್ಷೇ॒ ಸ॒ಪ್ತ ಹಸ್ತಾ᳚ಸೊ ಅಸ್ಯ । ತ್ರಿಧಾ᳚ ಬ॒ದ್ಧೋ ವೃ॑ಷ॒ಭೋ ರೋ᳚ರವೀತಿ ಮ॒ಹೋ ದೇ॒ವೋ ಮರ್ತ್ಯಾ॒ँ ಆವಿ॑ವೇಷ । ಅಗ್ನಿಮೂರ್ತಿ೦ ಧ್ಯಾಯಾಮಿ ।
  </ವಿಷಯ>
  <ವಿಷಯ
    ವಿನ್ಯಾಸ="ವಿನಿಯೋಗ"
  >
    ಓ೦ ಅಗ್ನಿ೦ ಪರ್ಯುಕ್ಷಣ ,ಪರಿಸಮೋಹನ , ಪರಿಷಿಚ್ಯ ।
  </ವಿಷಯ>
</View>
`;
